waveorder/models/inplane_oriented_thick_pol3d.py

```diff
@@ -98,12 +142,9 @@ def apply_inverse_transfer_function(
         *background_corrected_stokes
     )
 
-    # Return retardance in distance units (matching wavelength_illumination)
-    retardance = adr_parameters[0] * wavelength_illumination / (2 * np.pi)
-
     # Apply orientation transformations
     orientation = stokes.apply_orientation_offset(
         adr_parameters[1], rotate=rotate_orientation, flip=flip_orientation
     )
 
-    return retardance, orientation, adr_parameters[2], adr_parameters[3]
```

Comment:
Reviewers: this is the "critical" change.

I'm now skipping the multiplication by wavelength_illumination / (2 * np.pi)